sphinx/source/governance.rst

```diff
@@ -82,12 +82,16 @@ The details of pending proposals, including the proposer member ID, proposal scr
 
 In this case, there is one pending proposal (``id`` is 1), proposed by the first member (``member1``, ``id`` is 0) and which will call the ``accept_node`` function with the new member's certificate as a parameter. Two votes have been cast: ``member1`` (proposer) has voted for the proposal, while ``member2`` (``id`` is 1) has voted against it.
 
-Removing proposals
-------------------
+Withdrawing proposals
+---------------------
 
-At any stage during the voting process and before the proposal is completed, the proposing member may decide to remove a pending proposal:
+At any stage during the voting process and before the proposal is completed, the proposing member may decide to withdraw a pending proposal:
 
 .. code-block:: bash
 
-    $ memberclient --server-address 127.83.203.69:55526 --cert member1_cert.pem --privk member1_privk.pem --ca networkcert.pem removal --proposal-id 0
+    $ memberclient --server-address 127.83.203.69:55526 --cert member1_cert.pem --privk member1_privk.pem --ca networkcert.pem withdraw --proposal-id 0
     {"commit":110,"global_commit":109,"id":0,"jsonrpc":"2.0","result":true,"term":4}
+
+This means future votes will be ignored, and the proposal will never be accepted. However it will remain visible as a proposal so members can easily audit historic proposals.
```

Comment:
I think it's quite confusing to have both withdraw and remove, I would prefer keeping only withdraw. The only benefit of remove seems to be a tiny bit of ledger space saving, which is likely insignificant and does not make up for the confusion in my opinion.

Reply:
I think I agree. I'm wary of normalising arbitrary memory growth without _some_ way of removing state, but its probably not worth adding individual removes for every operation. We should consider a generic cull/remove call in future for network management, but until its a problem we shouldn't overcomplicate the interface. Will remove.